### PR TITLE
Add more event detail

### PR DESCRIPTION
**Pull request by castrojo**
This text comes for a previously authored document that was peer reviewed by sig-contribex.

### CODE REVIEW

**Comment by dims** [comment]
LGTM 👍

**Comment by jdumars** [comment]
LGTM

File: community/office-hours.md [hunk 1]
```diff
@@ -12,6 +12,36 @@ Third Wednesday of every month, there are two sessions:
...
+
+#### Bringing your own Stack Overflow Question
+
+If you submit a [SO question](https://stackoverflow.com/questions/tagged/kubernetes) we can prepare ahead of time and check important details. 
+
+As a thanks to the community the asker can then ensure that a well written answer makes it way to the question afterwards so that we can build a knowledge base and help maintain the incoming questions on SO. We can also use the video archive of each meeting to bring context to each SO question we answer. 
+
+Questions that aren’t addressed or need work can be punted to the next week or we can encourage other people to give them a look, at a bare minimum we can at least help socialize the difficult questions. 
+
+The hosts will do a shout out of thanks to the current leaderboard of SO answerers each meeting so the people helping answer questions can get some recognition.
```
Comment (spiffxp) [inline community/office-hours.md]: change current leaderboard to a link?

File: community/office-hours.md [hunk 2]
```diff
@@ -12,6 +12,36 @@ Third Wednesday of every month, there are two sessions:
...
 
+### How it Works
+
+#### Bringing your own Stack Overflow Question
+
+If you submit a [SO question](https://stackoverflow.com/questions/tagged/kubernetes) we can prepare ahead of time and check important details. 
+
+As a thanks to the community the asker can then ensure that a well written answer makes it way to the question afterwards so that we can build a knowledge base and help maintain the incoming questions on SO. We can also use the video archive of each meeting to bring context to each SO question we answer. 
+
+Questions that aren’t addressed or need work can be punted to the next week or we can encourage other people to give them a look, at a bare minimum we can at least help socialize the difficult questions.
```
Comment (spiffxp) [inline community/office-hours.md]: where can I go to see a running backlog of questions yet to be answered?

File: community/office-hours.md [hunk 3]
```diff
@@ -12,6 +12,36 @@ Third Wednesday of every month, there are two sessions:
 Tune into the [Kubernetes YouTube Channel](https://www.youtube.com/c/KubernetesCommunity/live) to follow along. 
 You can post questions on the [#office-hours channel](https://kubernetes.slack.com/messages/office-hours) on Slack.
 
+### How it Works
+
+#### Bringing your own Stack Overflow Question
+
+If you submit a [SO question](https://stackoverflow.com/questions/tagged/kubernetes) we can prepare ahead of time and check important details. 
+
+As a thanks to the community the asker can then ensure that a well written answer makes it way to the question afterwards so that we can build a knowledge base and help maintain the incoming questions on SO. We can also use the video archive of each meeting to bring context to each SO question we answer.
```
Comment (spiffxp) [inline community/office-hours.md]: where would the final log of question/answer go? meeting notes? .md file in a repo someplace?

File: community/office-hours.md [hunk 4]
```diff
@@ -12,6 +12,36 @@ Third Wednesday of every month, there are two sessions:
 Tune into the [Kubernetes YouTube Channel](https://www.youtube.com/c/KubernetesCommunity/live) to follow along. 
 You can post questions on the [#office-hours channel](https://kubernetes.slack.com/messages/office-hours) on Slack.
 
+### How it Works
+
+#### Bringing your own Stack Overflow Question
+
+If you submit a [SO question](https://stackoverflow.com/questions/tagged/kubernetes) we can prepare ahead of time and check important details.
```
Comment (spiffxp) [inline community/office-hours.md]: based on text above seems I'm assuming #office-hours is the only place to submit questions?

File: community/office-hours.md [hunk 5]
```diff
@@ -12,6 +12,36 @@ Third Wednesday of every month, there are two sessions:
...
+- Bad: How do I deploy my application in Kubernetes? (Easy to look up, tons of content on this) 
+- Good: What best practice techniques are there for deploying applications in Kubernetes? (Nuanced version lets us cover RBAC, application lifecycle)  
+- Bad: My pods aren’t coming up on a node. (Vague)
+- Good: My pods aren’t coming up or misbehaving, what are the debugging steps to find out the root cause? (Helps us communicate the thought process for debugging to a general audience) 
+
+#### What’s Offtopic
+
+Local installation and debugging: The participants don’t have access to your network or your hardware. The host can/should help the user transform a vague question into something answerable and reusable. 
+
+Let’s try to just not dismiss bad questions outright, but use it as an opportunity for the answer to be a teaching tool as opposed to just answering “It’s in /var/log/foo, next question.” If the question is about logging then the developers might as well share their experiences in that area, recommend tools, share an anecdote, things of that nature.
```
Comment (spiffxp) [inline community/office-hours.md]: s/just not/not just/

File: community/office-hours.md (hunk 3 above)
Comment (castrojo) [reply community/office-hours.md]: Ideally in the SO question itself.

**Pull request merged**